app/controllers/nonsipp/landorproperty/LandPropertyAddressManualController.scala

```diff
@@ -199,13 +199,19 @@ object LandPropertyAddressManualController {
       "landPropertyAddressManual.field5.error.max"
     )
 
+  private val postCodeFormErrors = InputFormErrors.postcode(
+    "landOrPropertyPostcodeLookup.postcode.error.required",
```

Comment:
Would it be worth replacing this with an empty string, like with fields 2 and 3 on lines 176 and 183? Or is there a particular reason why this is here but the others have been replaced with empty strings?